sickbeard/postProcessor.py

```diff
@@ -237,24 +237,25 @@ def recursive_glob(treeroot, pattern):
 
             # Add the extensions that the user doesn't allow to the 'extensions_to_delete' list
             if sickbeard.MOVE_ASSOCIATED_FILES:
-                allowed_extensions = sickbeard.ALLOWED_EXTENSIONS.split(",")
+                allowed_extensions = sickbeard.ALLOWED_EXTENSIONS.split(',')
                 found_extension = associated_file_path.rpartition('.')[2]
-                if not found_extension in allowed_extensions:
-                    self._log(u"Associated file extension not found in allowed extension {0}. Deleting it".format(found_extension), logger.WARNING)
+                if found_extension and not found_extension in allowed_extensions:
+                    self._log(u'Associated file extension not found in allowed extension: .{0}'
+                                .format(found_extension.upper()), logger.DEBUG)
```

Comment:
If you have a proper IDE with pylint enabled, it will align it for you. Else you'll have to do it by hand yes.